### PR TITLE
Fixed shellcheck findings in check_apt_upgrade

### PR DESCRIPTION
**Pull request by johannesst**
Fixed some shellcheck findings. Since check_apt still gives wrong results on my Focal and Jammy machines I think this plugin still have it's place

### CODE REVIEW

File: plugins/check_apt_upgrade [hunk 1]
```diff
@@ -94,8 +94,9 @@ EOF
 hwe_support_status () {
   [ -e "$PATH_TO_HWE_SUPPORT_STATUS" ] || return
 
-  HWE_SUPPORT_STATUS_MSG="; $($PATH_TO_HWE_SUPPORT_STATUS --$verbosity 2>&1)"
-  if [ "$?" != 0 ]; then
+  #HWE_SUPPORT_STATUS_MSG="; $($PATH_TO_HWE_SUPPORT_STATUS --$verbosity 2>&1)"
+  #if [ "$?" != 0 ]; then
```
Comment (simondeziel) [inline plugins/check_apt_upgrade:98]: Would you mind dropping the commented out version, please?

File: plugins/check_apt_upgrade [hunk 2]
```diff
@@ -113,8 +114,9 @@ fi
 # Note that we use `"$@"' to let each command-line parameter expand to a
 # separate word. The quotes around `$@' are essential!
 # We need TEMP as the `eval set --' would nuke the return value of getopt.
-TEMP=$($GETOPT -q -s sh -n "$SCRIPT_NAME" -o +hnvt: -l skip-hwe-check -- "$@")
-if [ $? != 0 ]; then
+#TEMP=$($GETOPT -q -s sh -n "$SCRIPT_NAME" -o +hnvt: -l skip-hwe-check -- "$@")
+#if [ $? != 0 ]; then
```
Comment (simondeziel) [inline plugins/check_apt_upgrade:118]: Same here too.

**Review by simondeziel — CHANGES_REQUESTED** [review]

File: plugins/check_apt_upgrade (hunk 2 above)
Comment (johannesst) [reply plugins/check_apt_upgrade]: Done m see  345efd4 :)

File: plugins/check_apt_upgrade (hunk 1 above)
Comment (johannesst) [reply plugins/check_apt_upgrade]: Done, see  345efd4 :)

**Review by simondeziel — APPROVED** [review]

**Pull request merged**